src/ui/src/builder/settings/BuilderFieldsWriterResourceId.vue

```diff
@@ -83,18 +89,21 @@ const linkTooltip = computed(() => {
 });
 
 const ressourceUrl = computed(() => {
-	if (!selected.value) return;
+	const value = Array.isArray(selected.value)
```

Comment:
I guess we can't display the resource URL if user selected multiple items. I think we should early return if `selected.value` is an array